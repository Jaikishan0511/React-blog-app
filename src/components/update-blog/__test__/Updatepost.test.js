import React from "react";
import renderer from "react-test-renderer";

import Updatepost from "../index";
describe("Updatepost", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Updatepost />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
