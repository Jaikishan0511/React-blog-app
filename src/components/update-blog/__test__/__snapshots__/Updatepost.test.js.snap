// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Updatepost snapshot renders 1`] = `
<div>
  <input
    onChange={[Function]}
    value=""
  />
  <br />
  <input
    onChange={[Function]}
    value=""
  />
  <br />
  <button
    onClick={[Function]}
  >
    Submit
  </button>
</div>
`;
